### ISSUE TITLE
Renew gittuf metadata

### ISSUE THREAD

**Issue by adityasaky** [gittuf/gittuf#281]
### Add a description

In conjunction with https://github.com/gittuf/gittuf/issues/280, we must add `gittuf trust renew` and `gittuf policy renew` subcommands to update the expiry (and version number) without changing anything else.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by adityasaky** [comment]
In working on this, we should think about expiry being bumped up during other metadata modifying actions. Should expiry be auto renewed when we add a new rule, for example?

**Comment by neilnaveen** [comment]
> In working on this, we should think about expiry being bumped up during other metadata modifying actions. Should expiry be auto renewed when we add a new rule, for example?

Isn't this already partially happening, whenever we add a new rule, the metadata version is being updating. I would think that when the version is updated it is confirmation that this metadata is good, and the expiry should be bumped up.

**Comment by adityasaky** [comment]
I tend to agree but I'd like to also hear if these are separated in TUF contexts. cc @JustinCappos

**Comment by JustinCappos** [comment]
expiry is mostly for freeze attack prevention and is mostly used to detect timestamp being replayed or on other top level roles to prevent an attack where the repo + timestamp is compromised.  If you have a repo where trusted keys are making changes, then I think this isn't that likely to be a concern in the gittuf case.  

However, I do think at some point we need to have some serious thought about this issue to see how we can do the best possible thing here.  This doesn't seem like a super high-impact threat vector regardless, so this issue seems low priority for now...
